centaur/src/main/resources/standardTestCases/ignore_noAddress.test

```diff
@@ -0,0 +1,14 @@
+# NB: To request this test by name, make it lowercase, eg sbt "centaur/it:testOnly * -- -n ignore_noaddress"
```

Comment:
Centaur has the right idea

Reply:
Yeah, I wondered whether you'd spot that!